caveclient/chunkedgraph.py

```diff
@@ -966,6 +995,8 @@ def get_lineage_graph(
             params.update(package_timestamp(timestamp_past, name="timestamp_past"))
         if timestamp_future is not None:
             params.update(package_timestamp(timestamp_future, name="timestamp_future"))
+        else:
```

Comment:
felt like lineage graph should not include things forward in time if the timestamp is set, thoughts?

Reply:
Agreed, that behavior would make sense to me.